initial checkin for blog (#345)
kubewarden/kubewarden.io: content/blog/2025/05/adopting-kubewarden.md

---
title: "Adopting of Kubewarden"
authors:
  - Robert Sirchia

> date: 2025-05-19

Reviewer: ```suggestion
date: 2025-05-23
```

---

## Call for Adopters

Kubewarden is showing significant maturity as a Kubernetes policy enforcement solution, with a growing number of organizations adopting it for policy enforcement for their clusters. This trend reflects the increasing need for robust, flexible, and auditable policy enforcement in the Kubernetes ecosystem.

## Why Kubewarden?

> But why use Kubewarden? Kubewarden has seen a substantial expansion of its policy library. More pre-built policies are available, covering a wider range of security and operational best practices. Kubewarden's use of WebAssembly (Wasm) allows for policies to be written in various languages, providing developers with flexibility. WASM offer high performance and efficient resource utilization, which is crucial in Kubernetes environments. With the ability to verify policies using SLSA (Supply Chain Levels for Software Artifacts).

Reviewer 2: ```suggestion
But why use Kubewarden? Kubewarden has seen a substantial expansion of its policy library. More pre-built policies are available, covering a wider range of security and operational best practices. Kubewarden's use of WebAssembly (Wasm) allows for policies to be written in various languages, providing developers with flexibility. Wasm offers high performance and efficient resource utilization, which is crucial in Kubernetes environments. Kubewarden can verify policies using SLSA (Supply Chain Levels for Software Artifacts).
```

> But why use Kubewarden? Kubewarden has seen a substantial expansion of its policy library. More pre-built policies are available, covering a wider range of security and operational best practices. Kubewarden's use of WebAssembly (Wasm) allows for policies to be written in various languages, providing developers with flexibility. WASM offer high performance and efficient resource utilization, which is crucial in Kubernetes environments. With the ability to verify policies using SLSA (Supply Chain Levels for Software Artifacts).

Reviewer 3: Maybe we can be more specific by mentioning our [support for CEL](https://www.kubewarden.io/blog/2024/06/welcome-cel-policy/), flexibility for operators via the self-service PolicyServer and PolicyGroups CRDs, and providing compliance cronjob checks beyond just admission control.
